### PR TITLE
Add Enum & Bounded to exports

### PR DESCRIPTION
**Pull request by miymiy**

### CODE REVIEW

**Comment by domdere** [comment]
@miymiy Hmm could we create a new module `Preamble.Unsafe` and put the `Enum` export in there pls? (copy the header from the `Preamble` module pls.)

**Comment by miymiy** [comment]
OK

File: src/Preamble/Unsafe.hs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+-- |
+-- Module       : Preamble.Unsafe
+-- Copyright    : (C) 2014-2015 Dom De Re
+-- License      : BSD3
+-- Maintainer   : Dom De Re
+--
+-- The unsafe modules from Prelude
+--
+-------------------------------------------------------------------
+
```
Comment (domdere) [inline src/Preamble/Unsafe.hs]: sorry this is a nitpick, but can we get rid of this blank line pls?

Comment (domdere) [reply src/Preamble/Unsafe.hs]: good to go in after that on a 🤢 build

Comment (miymiy) [reply src/Preamble/Unsafe.hs]: 🌵

**Comment by domdere** [comment]
@miymiy thanks!

**Pull request merged**